src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -598,4 +602,79 @@ public void publishEvent(final Integer procId, final String eventId,
     }).collect(Collectors.toList());
   }
 
+  /**
+   * Returns a list of message summary at the event level.
+   *
+   * @param procId
+   * @param eventId
+   * @param messageDirection
+   * @param messageRead
+   * @param sort
+   * @param page
+   * @param pageSize
+   * @return
+   */
+  public Collection<MessageSummary> getMessageSummaries(
```

Comment:
As with the controller - probably be good to start a new `MessagingService` or similar - perhaps liaise with @venkibric as he might have already done so 👍

Reply:
Yep - agree this would be good 👍

Reply:
Venki created those, need to wait for his PR into develop